resumes-recherche/index.md

```diff
@@ -2,12 +2,12 @@
 altLangPage: "https://design.canada.ca/research-summaries/"
 date: 2024-01-17
 dateModified: 2024-01-17
-description: ""
+description: "Résumés des recherches menées par le Bureau de la transformation numérique (BTN) et par les ministères fédéraux."
 language: fr
 title: "Résumés de recherche"
 ---
-<p>Summaries of the research the Digital Transformation Office (DTO) and federal departments undertook to make it easier for people to find and understand Government of Canada information and services on Canada.ca.</p>
-<h2>List of research summaries</h2>
+<p>Résumés des recherches menées par le Bureau de la transformation numérique (BTN) et par les ministères fédéraux. Le but de cette recherche était d’aider les gens à trouver et à comprendre les renseignements et services offerts par le gouvernement du Canada sur Canada.ca</p>
+<h2>Liste de résumés de recherche</h2>
 <ul class="mrgn-tp-lg">
```

Comment:
Can we explore a dynamic way to update this list? Like a for loop that returns all items in the resumes-recherches folder.

Reply:
Unless we don't foresee any new additions to this list any time soon.

Reply:
Updates will be few and not for a while.  I would suggest updating this to something more dynamic if it becomes unwieldy.